WooCommerce/Classes/ServiceLocator/PushNotesManager.swift

```diff
@@ -7,7 +7,7 @@ protocol PushNotesManager {
     /// An observable that emits values when the Remote Notifications are received while the app is
     /// in the foreground.
     ///
-    var foregroundNotifications: Observable<ForegroundNotification> { get }
+    var foregroundNotifications: Observable<PushNotification> { get }
```

Comment:
I'm sure everything will make sense in a later PR, just wondering why this `foregroundNotifications` emits a `PushNotification` while `inactiveNotifications` emits a `ForegroundNotification` from the PR description? or did you mean that `inactiveNotifications` will also emit a `PushNotification`?

Reply:
> while inactiveNotifications emits a ForegroundNotification from the PR description? 

My bad. Copy-paste mistake. 🤪 Yep, the PR description should have said `PushNotification`. I fixed it.